<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>888</width>
    <height>440</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QFrame" name="frame">
        <property name="styleSheet">
         <string notr="true">border-color: rgb(85, 0, 127);</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_5">
         <item row="0" column="0">
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0">
            <widget class="QFrame" name="frame_2">
             <property name="styleSheet">
              <string notr="true">background-color: rgb(255, 255, 207);
color: rgb(0, 0, 0);</string>
             </property>
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <widget class="QLabel" name="label">
              <property name="geometry">
               <rect>
                <x>220</x>
                <y>10</y>
                <width>319</width>
                <height>25</height>
               </rect>
              </property>
              <property name="font">
               <font>
                <pointsize>16</pointsize>
                <stylestrategy>PreferAntialias</stylestrategy>
               </font>
              </property>
              <property name="text">
               <string>EDITAR PRODUTO SELECIONADO</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
             <widget class="QPushButton" name="salvar">
              <property name="geometry">
               <rect>
                <x>350</x>
                <y>360</y>
                <width>65</width>
                <height>23</height>
               </rect>
              </property>
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>50</width>
                <height>20</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>150</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>14</pointsize>
               </font>
              </property>
              <property name="cursor">
               <cursorShape>PointingHandCursor</cursorShape>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
border-radius:10;
alineItens:center;
</string>
              </property>
              <property name="text">
               <string>SALVAR</string>
              </property>
             </widget>
             <widget class="QWidget" name="layoutWidget">
              <property name="geometry">
               <rect>
                <x>11</x>
                <y>200</y>
                <width>830</width>
                <height>22</height>
               </rect>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout">
               <item>
                <widget class="QLineEdit" name="ln_cod">
                 <property name="text">
                  <string/>
                 </property>
                 <property name="placeholderText">
                  <string>CÓDIGO</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="ln_desc">
                 <property name="text">
                  <string/>
                 </property>
                 <property name="placeholderText">
                  <string>DESCRIÇÃO</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="ln_dia">
                 <property name="placeholderText">
                  <string>PREÇO DIÁRIA</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="ln_mes">
                 <property name="placeholderText">
                  <string>PREÇO MENSAL</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="ln_mes_2">
                 <property name="placeholderText">
                  <string>PREÇO VENDA</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
